Slim/Player/TranscodingHelper.pm

```diff
@@ -197,7 +200,11 @@ sub _getCapabilities {
 
 		if ($capabilities =~ /^:\{(\w+=[^}]+)\}/) {
 			$capabilities = $';
-			$args = $1;
+			if ($can eq 'E') {
+				$args = { split (/\s*[=,]\s*/, $1) };
```

Comment:
Wow... you're picking up Perl stuff I've never thought of before! Nice (if it's actually working 😁)!

Reply:
It seems to ... at least for the few times I've tried, the hash has keys with space trimmed. I know I'm a bit extreme trying to find the most compact expression and Perl is full of temptations.